### PR TITLE
pythonPackages.simanneal: init at 0.4.1

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] Linux
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/simanneal/default.nix [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+buildPythonPackage rec {
+  version = "0.4.1";
+  name = "simanneal-${version}";
+
+  src = fetchurl {
+    url = "mirror://pypi/s/simanneal/${name}.tar.gz";
+    sha256 = "1ib1hv2adq6x25z13nb4s1asci6dqarpyhpdq144y2rqqff2m01x";
+  };
+
+  meta = {
```
Comment (vyp) [inline pkgs/development/python-modules/simanneal/default.nix]: A maintainer must be set, did you want to maintain this package?

**Review by vyp — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/simanneal/default.nix [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{ lib, fetchurl, buildPythonPackage }:
+
+buildPythonPackage rec {
+  version = "0.4.1";
+  name = "simanneal-${version}";
+
+  src = fetchurl {
```
Comment (FRidh) [inline pkgs/development/python-modules/simanneal/default.nix]: fetchPypi

File: pkgs/development/python-modules/simanneal/default.nix [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{ lib, fetchurl, buildPythonPackage }:
+
+buildPythonPackage rec {
+  version = "0.4.1";
```
Comment (FRidh) [inline pkgs/development/python-modules/simanneal/default.nix]: could you use `pname` here

**Review by FRidh — CHANGES_REQUESTED** [review]
are tests working?

**Comment by veprbl** [comment]
@FRidh tests are not distributed via PyPI currently. I made a PR to address that: perrygeo/simanneal#21

**Pull request merged**